.github/workflows/lintandtest.yml

```diff
@@ -9,16 +13,23 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
-    - uses: eshork/gitleaks-action@master
     - name: Set up Python 3.8
       uses: actions/setup-python@v1
       with:
-        python-version: 3.8
+        python-version: '3.8'
+
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt
+
     - name: Lint with pylint
-      run: pylint -E nachomemes
+      run: |
+        pip install pylint
+        pylint -E *.py
+        
     - name: Test with unittest
-      run: python3 -m unittest
+      run: |
+        python3 -m unittest
+    - name: Build Docker Image
+      run: docker build . 
```

Comment:
Do we need to name/tag the image here?

Reply:
No this is basically just testing that that it builds, not pushing at this step